vcluster/configure/vcluster-yaml/external/platform/api-key.mdx

```diff
@@ -2,28 +2,262 @@
 title: apiKey
 sidebar_label: apiKey
 sidebar_position: 1
+toc_max_heading_level: 2
 ---
 
 import ProAdmonition from '../../../../_partials/admonitions/pro-admonition.mdx'
 
+import Mark from "@site/src/components/Mark";
+import InterpolatedCodeBlock from "@site/src/components/InterpolatedCodeBlock";
 import ApiKey from '@site/platform/api/_partials/resources/config/external/platform/apiKey.mdx'
+import Flow, { Step } from '@site/src/components/Flow';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
 
 <ProAdmonition/>
 
-This provides a way to connect directly to the vCluster Platform.
+<!--vale off-->
+
+Enable secure communication between your virtual cluster and the vCluster Platform. 
+You can configure the platform to manage and authenticate virtual cluster connections consistently across different environments.
+
+## Connect virtual clusters to the vCluster Platform 
+
+To connect a virtual cluster to the vCluster Platform, you can use an API key—referred to as an [access key](/platform/next/api/authentication) in the platform's console. There are two authentication methods:
+
+- **Shared access** (_Recommended_): Create a single Kubernetes Secret containing the access key, and reference it in multiple virtual clusters, even if they are deployed in different namespaces. This approach simplifies management, reduces overhead, and eases credential rotation, making it suitable for most users and production environments. For most scenarios, using a shared access key is the preferred approach due to its simplicity and ease of maintenance.
+
+- **Individual user access**: Create separate Kubernetes Secrets, each with a unique personal access key, for every virtual cluster. This method provides granular access control and allows revocation of access to individual clusters.
```

Comment:
nit: I wouldn't call it "user" access, since there is no individual in this. Maybe we use something like "Shared access key" and "Dedicated access key" terms?